### PR TITLE
First commit of the Automation site

### PR DESCRIPTION
**Pull request by steven7324**

### CODE REVIEW

File: src/test/java/com/automation/test/BuyPracticeTest.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+
+import java.time.Duration;
+import java.util.NoSuchElementException;
+import java.util.concurrent.TimeUnit;
+
+
+public class BuyPracticeTest {
```
Comment (aperdomob) [inline src/test/java/com/automation/test/BuyPracticeTest.java]: can you explain to me why did you name the class like thtat

File: src/test/java/com/automation/test/BuyPracticeTest.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
+        driver.get("http://automationpractice.com/");
+        //addToCart and Checkout
+        WebElement btnViewCloth = driver.findElement(By.xpath("//ul[@id='homefeatured']/li[1]/div"));
+
+        //Scroll to Element
+        JavascriptExecutor js = (JavascriptExecutor) driver;
+        js.executeScript("arguments[0].scrollIntoView(true);", btnViewCloth);
+        wait.until(ExpectedConditions.visibilityOf(btnViewCloth));
+        actions.moveToElement(btnViewCloth).perform();
```
Comment (aperdomob) [inline src/test/java/com/automation/test/BuyPracticeTest.java]: why do you need to do scroll

**Review by aperdomob — CHANGES_REQUESTED** [review]
keep the Google test

File: src/test/java/com/automation/test/BuyPracticeTest.java (hunk 1 above)
Comment (steven7324) [reply src/test/java/com/automation/test/BuyPracticeTest.java]: I just put that name. I was thinking to change it for BuyClothesTest. Would It be better in that way or there is some convention to name it?

File: src/test/java/com/automation/test/BuyPracticeTest.java (hunk 2 above)
Comment (steven7324) [reply src/test/java/com/automation/test/BuyPracticeTest.java]: That's scroll it's needed to run the automation with firefox

File: src/test/java/com/automation/test/BuyPracticeTest.java (hunk 1 above)
Comment (aperdomob) [reply src/test/java/com/automation/test/BuyPracticeTest.java]: change to BuyClothesTest

File: src/test/java/com/automation/test/BuyPracticeTest.java (hunk 2 above)
Comment (aperdomob) [reply src/test/java/com/automation/test/BuyPracticeTest.java]: use scroll into your tests add flakiness, please try to find a solution without scroll

**Review by aperdomob — CHANGES_REQUESTED** [review]

**Review by aperdomob — APPROVED** [review]

**Pull request merged**